README.md

```diff
@@ -296,3 +312,19 @@ automatically map legacy groups to Keycloak groups, by specifying the mapping in
 
 This switch can be toggled to decide whether groups which are not defined in the legacy group conversion map should be
 migrated anyway or simply ignored.
+
+## Totp
+This module supports the migration of totp devices. The totp configuration block looks like this:
+```json
+{
+  "name": "Totp Device 1",
+  "secret": "secret",
+  "digits": 6,
+  "period": 30,
+  "algorithm": "HmacSHA1",
+  "encoding": "BASE32"
+}
+```
+name should be the name of the totp device, while secret is the secret, that could be encoded in "BASE32" or as the utf8 bytes. 
```

Comment:
Using `backticks` for the properties would increase readability. E.g.,:

"`name` should be the name of the totp device (...)".